packages/postcss-unique-selectors/src/index.js

```diff
@@ -1,11 +1,22 @@
 import { plugin } from 'postcss';
 import sort from 'alphanum-sort';
 import uniqs from 'uniqs';
+import selectorParser from 'postcss-selector-parser';
+
+function parseSelectors(selectors, callback) {
+  return selectorParser(callback).processSync(selectors);
+}
 
 function unique(rule) {
   rule.selector = sort(uniqs(rule.selectors), { insensitive: true }).join();
 }
 
 export default plugin('postcss-unique-selectors', () => {
-  return (css) => css.walkRules(unique);
+  return (css) =>
+    css.walkRules((nodes) => {
+      nodes.selector = parseSelectors(nodes.selector, (selNode) => {
+        selNode.walk((sel) => (sel.type === 'comment' ? sel.remove() : sel));
```

Comment:
We should not remove comment, it is bad idea, this plugin should keep comments

Reply:
I cant keep the original position of the comments after sorting, I can append it at the end.
is it fine ?

Reply:
Yes, we should not remove comments

Reply:
Okay will change it 👍